cmd/fs-v1-metadata.go

```diff
@@ -315,37 +317,63 @@ func createFormatFS(fsPath string) error {
 }
 
 func initFormatFS(fsPath string) (rlk *lock.RLockedFile, err error) {
+	// Done channel is used to close any lingering retry routine, as soon
```

Comment:
They can't be added.. because you would require a separate process, - this will be taken care in our functional tests.